### ISSUE TITLE
Problem with neural network

### ISSUE THREAD

**Issue by maksimliubimov** [pseudoyim/galvaneye#2]
Hello, can you help me? I tried to learn neural network but I have this result: Using TensorFlow backend.
/usr/local/lib/python2.7/dist-packages/sklearn/cross_validation.py:44: DeprecationWarning: This module was deprecated in version 0.18 in favor of the model_selection module into which all the refactored classes and functions are moved. Also note that the interface of the new CV iterators are different from that of this module. This module will be removed in 0.20.
  "This module will be removed in 0.20.", DeprecationWarning)
Loading training data...
['train_labels', 'train']
(424, 38400)
(424, 3)
Shape of feature array:  (424, 38400)
Shape of label array:  (424, 3)
Total time taken to load image data: 0.713379859924 seconds
Training...
Train on 339 samples, validate on 85 samples
Epoch 1/20
339/339 [==============================] - 0s - loss: 1.1011 - acc: 0.3186 - val_loss: 1.0821 - val_acc: 0.4588
Epoch 2/20
339/339 [==============================] - 0s - loss: 1.0958 - acc: 0.3333 - val_loss: 1.0769 - val_acc: 0.5412
Epoch 3/20
339/339 [==============================] - 0s - loss: 1.0874 - acc: 0.3982 - val_loss: 1.0711 - val_acc: 0.4941
Epoch 4/20
339/339 [==============================] - 0s - loss: 1.0786 - acc: 0.4602 - val_loss: 1.0648 - val_acc: 0.5176
Epoch 5/20
339/339 [==============================] - 0s - loss: 1.0733 - acc: 0.4926 - val_loss: 1.0591 - val_acc: 0.4941
Epoch 6/20
339/339 [==============================] - 0s - loss: 1.0589 - acc: 0.4985 - val_loss: 1.0538 - val_acc: 0.4941
Epoch 7/20
339/339 [==============================] - 0s - loss: 1.0525 - acc: 0.4985 - val_loss: 1.0494 - val_acc: 0.4941
Epoch 8/20
339/339 [==============================] - 0s - loss: 1.0420 - acc: 0.4985 - val_loss: 1.0461 - val_acc: 0.4941
Epoch 9/20
339/339 [==============================] - 0s - loss: 1.0414 - acc: 0.5015 - val_loss: 1.0441 - val_acc: 0.4941
Epoch 10/20
339/339 [==============================] - 0s - loss: 1.0314 - acc: 0.5015 - val_loss: 1.0431 - val_acc: 0.4941
Epoch 11/20
339/339 [==============================] - 0s - loss: 1.0239 - acc: 0.5044 - val_loss: 1.0429 - val_acc: 0.4941
Epoch 12/20
339/339 [==============================] - 0s - loss: 1.0169 - acc: 0.5015 - val_loss: 1.0433 - val_acc: 0.4941
Epoch 13/20
339/339 [==============================] - 0s - loss: 1.0154 - acc: 0.5015 - val_loss: 1.0440 - val_acc: 0.4941
Epoch 14/20
339/339 [==============================] - 0s - loss: 1.0181 - acc: 0.5015 - val_loss: 1.0447 - val_acc: 0.4941
Epoch 15/20
339/339 [==============================] - 0s - loss: 1.0019 - acc: 0.5015 - val_loss: 1.0451 - val_acc: 0.4941
Epoch 16/20
339/339 [==============================] - 0s - loss: 1.0068 - acc: 0.5015 - val_loss: 1.0448 - val_acc: 0.4941
Epoch 17/20
339/339 [==============================] - 0s - loss: 1.0000 - acc: 0.5015 - val_loss: 1.0446 - val_acc: 0.4941
Epoch 18/20
339/339 [==============================] - 0s - loss: 0.9944 - acc: 0.5015 - val_loss: 1.0434 - val_acc: 0.4941
Epoch 19/20
339/339 [==============================] - 0s - loss: 0.9801 - acc: 0.5015 - val_loss: 1.0419 - val_acc: 0.4941
Epoch 20/20
339/339 [==============================] - 0s - loss: 0.9913 - acc: 0.5015 - val_loss: 1.0399 - val_acc: 0.4941

Total time taken to train model: 5.78856301308 secucationds

Evaluation of model on test holdout set:
85/85 [==============================] - 0s

Loss score:  1.03991830349
Accuracy score:  0.494117647409
 Also training was instantaneous. It would be really good if you give me a question. Thank you

**Comment by AiBuddy** [comment]
@LyubimovMaxim  i wonder if u ever got ur answer

**Comment by muskansingla12** [comment]
@LyubimovMaxim can you provide me the dataset @muskansingla1999@gmail.com?
